tutor4points/core/templates/home.html

```diff
@@ -70,13 +100,88 @@ <h2>Suggested Sessions</h2>
                         {% endfor %}
                 </div>
             </div>
+            <div class="dashboard-container">
+                <div class="list-container">
+                    <h2>Sent Requests</h2>
+                    {% for sent_request in sent_requests|slice:":3" %}
+                        {% with tutor=sent_request.tutor %}
+                        <div class="info-container d-flex" id="tutor-{{tutor.id}}" data-tutorid="{{tutor.id}}" >
+                            <div>
+                                <img src="{{sent_request.tutor.profile_pic.url}}" class="profile-pic">
+                            </div>
+                            <div class="name-column">
+                                {{ tutor }} <br>
+                                <div class="classes-column">
+                                    {{ sent_request.class_name }} @ {{ sent_request.location }} <br/>
+                                </div>
+                            </div>
+                            <div class="times-column">
+                                {{sent_request.tutor_date}} @ {{sent_request.tutor_time}} <br/>
+                            </div>
+                            {% if sent_request.tutee_comment %}
+                                Comment: {{sent_request.tutee_comment}} <br> <br>
+                            {% else %}
+                                No comment <br> <br>
+                            {% endif %}
+                        </div>
+                        {% endwith %}
+                    {% endfor %}
+                </div>
+            </div>
+
+        </div>
+    </div>
+</div>
+
+{% if form.errors %}
+<script>
+    $(document).ready (function(){
+        $("#tutor-request-modal").modal('show')
+    })
+</script>
+{% endif %}
+
+
+<div class = "modal fade" id = "tutor-request-modal">
```

Comment:
This modal does not appear when clicking on the "request" button in the Suggested Sessions

Reply:
The request functionality seems to be working fine for me. Can someone else double check on this?

Reply:
I just double checked, and it also doesn't open the request modal but only links to the tutors page.